ui/admin/tests/acceptance/credential-store/create-test.js

```diff
@@ -101,23 +105,26 @@ module('Acceptance | credential-stores | create', function (hooks) {
       'credential-stores'
     ] = [];
     await visit(urls.credentialStores);
+
     assert.notOk(
       instances.scopes.project.authorized_collection_actions[
         'credential-stores'
       ].includes('create'),
     );
-    assert.notOk(find(`[href="${urls.newCredentialStore}"]`));
+    assert.dom(commonSelectors.HREF(urls.newCredentialStore)).doesNotExist();
   });
 
-  test('saving a new credential store with invalid fields displays error messages', async function (assert) {
+  test('saving a new static credential store with invalid fields displays error messages', async function (assert) {
+    const errorMsg =
+      'Invalid request. Request attempted to make second resource with the same field value that must be unique.';
```

Comment:
should we also move this error message to a common file considering we have this in all create-test files?

Reply:
This specific error is only used here. I have considered it for line 154 `The request was invalid.`. However, I have decided not to especially since Carlos brought out a good point of updating those tests to be more accurate. Most of them "fail" on the name field but that is only valid for Targets. Everything else needs to be changed to "fail" on a truly required field.

Reply:
All that to say, maybe this is something we address at a later time?